showpass/Showpass-E2E-tests
# TA-54 Verify the 'Credits' page appearance

**Description:**
This spec will be verifying the ‘Credits’ page overall appearance.
Steps:
- Navigate to the following page after logging in https://beta.showpass.com/account/credits/
- Verify that the page content is correct and consistent
- See the screenshot for reference

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-54

**How to test:**
Run the following test suite and ensure it passes
cypress/e2e/user-settings/credits.cy.js

**Test execution results:**

<img width="959" alt="TA-54-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/02f5ec60-d710-42da-98bb-6e8b667a3eb6">
